transmart-core-db/src/main/groovy/org/transmartproject/db/tree/TreeNode.groovy

```diff
@@ -268,4 +276,26 @@ class TreeNode {
         }
     }
 
+    final String getDimension() {
+        switch (tableName) {
+            case 'concept_dimension':
+                return DimensionImpl.CONCEPT.name
+            case 'patient_dimension':
+                return DimensionImpl.PATIENT.name
+            case 'modifier_dimension':
```

Comment:
@forus Specific patients (what you call "dimension instances") are called "dimension elements" in the design and hypercube code.